app/views/applications/deploy.html.erb

```diff
@@ -0,0 +1,93 @@
+<% content_for :page_title, @application.name %>
+
+<div class="page-header">
+  <h1 class="remove-bottom-margin">
+    <span class="name">Deploy <%= @application.name %></span>
+    <span class="shortname">(<%= @application.shortname %>)</span>
+  </h1>
+  <%= link_to @application.repo_url, @application.repo_url, target: "_blank" %>
+</div>
+
+<main>
+  <h2>Candidate Release <span class="label label-info"><%= @release_tag %></span></h2>
+  <p class="lead add-top-margin">Production was deployed to <span class="label label-danger"><%= @production_deploy.version %></span><%= time_ago_in_words(@production_deploy.created_at) %> ago</p>
```

Comment:
The phrasing on this feels off: I always think of this as a version being deployed to production, not production being deployed to a version. I would suggest rephrasing this to “`release_x` was deployed to production _n_ days ago”, but I’m guessing the alignment of the two tags is intentional. Not sure what other options there are.